ui/src/tests/beaconchaApi/testFetchValidatorIndex.ts

```diff
@@ -0,0 +1,47 @@
+import { BeaconchaApi } from "../../apis/beaconchaApi";
+import { expect } from "chai";
+import { beaconchaApiParamsMap } from "../../params";
+import { Web3signerGetResponse } from "../../apis/web3signerApi/types";
+import { networkTestMap } from "./networkTestMap";
+
+describe("Test for fetching validator indexes in every available network", () => {
+  it("should return data corresponding to every validator PK", () => {
+    const networks = ["mainnet", "prater", "gnosis"];
+
+    for (const network of networks) {
+      const keystoresGet = {
+        status: "ok",
+        data: [
+          {
+            validating_pubkey: networkTestMap.get(network)!.pubkeys[0],
```

Comment:
Ideally you should implement integration tests, setting up a web3signer and using the we3signer API for the tests. It could be done also for the consensus client API.

This way you are not hardcoding the response which may change in the future